### ISSUE TITLE
Crash when long pressing app icon

### ISSUE THREAD

**Issue by Zensonyx** [MM2-0/Kvaesitso#806]
## Description

When holding down on an icon and tapping back to the app drawer, Kvaesitso crashes and relaunches. This also happens sometimes when changing icons.

## Strack trace

```
java.lang.IllegalArgumentException: red = 0.0, green = 0.0, blue = 0.0, alpha = -3.8146972E-8 outside the range for sRGB IEC61966-2.1 (id=0, model=Rgb)
	at androidx.compose.ui.graphics.InlineClassHelperKt.throwIllegalArgumentException(InlineClassHelper.kt:3)
	at androidx.compose.ui.graphics.ColorKt.Color(Color.kt:17)
	at androidx.compose.ui.graphics.Color.copy-wmQWz5c$default(Color.kt:17)
	at de.mm20.launcher2.ui.launcher.search.common.grid.GridItemKt$ItemPopup$3.invoke(GridItem.kt:54)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:7)
	at androidx.compose.runtime.ComposerImpl.recomposeToGroupEnd(Composer.kt:585)
	at androidx.compose.runtime.ComposerImpl.skipCurrentGroup(Composer.kt:137)
	at androidx.compose.runtime.ComposerImpl.doCompose(Composer.kt:242)
	at androidx.compose.runtime.ComposerImpl.recompose$runtime_release(Composer.kt:30)
	at androidx.compose.runtime.CompositionImpl.recompose(Composition.kt:21)
	at androidx.compose.runtime.Recomposer.access$performRecompose(Recomposer.kt:77)
	at androidx.compose.runtime.Recomposer$runRecomposeAndApplyChanges$2$1.invoke(Recomposer.kt:545)
	at androidx.compose.ui.platform.AndroidUiFrameClock$withFrameNanos$2$callback$1.doFrame(AndroidUiFrameClock.android.kt:7)
	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.doFrame(AndroidUiDispatcher.android.kt:48)
[15 lines truncated]
```

## Device info
APP.VERSION : 1.30.0<br>APP.VERSIONCODE : 2024050700<br>APP.SIGNATURE : rx1fSnL7r5/OMoFC0e1KPqTndXQ=<br>LAUNCHER.APP : android<br>TIMEZONE : Europe/London<br>VERSION.RELEASE : 14<br>VERSION.INCREMENTAL : 11480754<br>VERSION.SDK.NUMBER : 34<br>BOARD : cheetah<br>BOOTLOADER : cloudripper-14.4-11458464<br>BRAND : google<br>CPU_ABI : arm64-v8a<br>CPU_ABI2 : <br>DISPLAY : AP1A.240405.002<br>FINGERPRINT : google/cheetah/cheetah:14/AP1A.240405.002/11480754:user/release-keys<br>HARDWARE : cheetah<br>HOST : abfarm-release-rbe-32-2004-00125<br>ID : AP1A.240405.002<br>MANUFACTURER : Google<br>MODEL : Pixel 7 Pro<br>PRODUCT : cheetah<br>TAGS : release-keys<br>TIME : 1708561357000<br>TYPE : user

**Comment by exstntlmsnthrp** [comment]
@Zensonyx Hmmm, I have the same device and I'm unable to duplicate either of those problems. Which version of Android are you using? 

It likely won't help, but have you tried going to ```Settings > Debug > Clean up database```? Or ```Settings > Debug > Reinstall icon packs```?

**Comment by MM2-0** [comment]
Surely this is fixed in 1.30.1 ![clueless](https://cdn.7tv.app/emote/6154ecd36251d7e000db18a0/1x.webp)
